osgar/drivers/realsense.py

```diff
@@ -0,0 +1,42 @@
+"""
+  RealSense2 OSGAR wrapper
+
+"""
+import pyrealsense2 as rs
+
+from osgar.node import Node
+from osgar.bus import BusShutdownException
+
+
+class RealSense(Node):
+    def __init__(self, config, bus):
+        super().__init__(config, bus)
+        bus.register('pose2d')
+        self.verbose = config.get('verbose', False)
+        self.pipe = None  # not initialized yet
+
+    def update(self):
+        channel = super().update()  # define self.time
+        if channel == 'trigger':
+            frames = self.pipe.wait_for_frames()
+            pose = frames.get_pose_frame()
+            if pose:
+                data = pose.get_pose_data()
+                x = data.translation.z
+                y = data.translation.x  # not sure yet
+                self.publish('pose2d', [int(x*1000), int(y*1000), 0])  # TODO heading
+        return channel
+
+    def run(self):
+        self.pipe = rs.pipeline()
+        cfg = rs.config()
```

Comment:
Let's enable just the position stream here so we are not needlessly loading the USB bus.